core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -441,6 +450,7 @@ public String toString() {
           .setDescription("Version of Alluxio. User should never modify this property.")
           .setIgnoredSiteProperty(true)
           .setIsHidden(true)
+          .setScope(Scope.ALL)
```

Comment:
could this be `Scope.NONE`?

Reply:
The same as I talked before, not sure the gap between Scope.NONE and ALL. 
In my criteria, this properties used in everywhere, so it falls into the category of ALL

Reply:
My worry is that in the case of client/server version mismatch, that the client would get this value from cluster defaults and subsequently report that it is the same version as the server - which may not be the case.

Reply:
Oh, that's possible, then this property should not be propagated to client or worker. Will change it to None

Reply:
The test failed because of they check VERSION on the client side, if it's different from the current one, this property will be removed and not merged into client/worker configuration.
I changed the Scope back to ALL

Reply:
oh :(